demos/python_demos/multi_camera_multi_person_tracking/README.md

```diff
@@ -32,16 +32,22 @@ and then for each detected object it extracts embeddings using re-identification
 To install required dependencies run
 
 ```bash
-pip3 install -r requirements.txt
+cat requirements.txt | xargs -n 1 -L 1 pip3 install
 ```
 
 ### Command line arguments
 
 Run the application with the `-h` option to see the following usage message:
 
 ```
-usage: multi_camera_multi_person_tracking.py [-h] -i I [I ...] -m M_DETECTOR
+usage: multi_camera_multi_person_tracking.py [-h] -i I [I ...]
+                                             [--detections DETECTIONS [DETECTIONS ...]]
+                                             -m M_DETECTOR
                                              [--t_detector T_DETECTOR]
+                                             [--m_segmentation M_SEGMENTATION]
+                                             [--t_segmentation T_SEGMENTATION]
+                                             [--m_orientation M_ORIENTATION]
+                                             [--t_orientation T_ORIENTATION]
```

Comment:
You need to expand the README with an explanation of these new parameters. In particular, it's necessary to note what models are accepted for the new `--m` options, as well as the format of the file(s) that `--detections` accepts.

Reply:
Model specified via `--m_orientation` and `--t_orientation` arguments removed. Instance segmentation model was added in readme (also is present in `models.lst` file).